src/Enzo/enzo_EnzoInitialM1Closure.hpp

```diff
@@ -0,0 +1,63 @@
+// See LICENSE_CELLO file for license and copyright information
+
+/// @file     enzo_EnzoInitialM1Closure.hpp
+/// @author   William Hicks (whicks@ucsd.edu)
+/// @date     06 September 2022
+/// @brief    [\ref Enzo] Initialization routine for EnzoMethodM1Closure.
+//                        This is needed to compute and store large-ish tables relevant
+//                        to EnzoMethodM1Closure that shouldn't be stored as attributes
+//                        of the EnzoBlock class (e.g. Reading in HLL eigenvalue tables and cross sections
+//                          for a variety of different stellar masses).
+//
+//                        These things can't be stored as attributes of EnzoMethodM1Closure
+//                        because attributes of the derived class get sliced away after
+//                        refresh.      
```

Comment:
I think you're right about this. I originally had the iterator for group number stored as an attribute that would get updated throughout the method, which resulted in a segfault after the cast from `Method*` to `EnzoMethodM1Closure*`, and assumed that the error would also happen to the table HLL if I read the file in the constructor and stored it as an attribute.

I'll try fixing this. Just realized that attaching the HLL table to an initializer may also result in some issues with restarting from a checkpoint

Reply:
If you're having trouble just let me know - as I mentioned (I tested this out on a local branch and there didn't seem to be any issues)

If you haven't gotten to this already, I had a quick organizational suggestion:
* It might be helpful to create a custom class called something like ``M1HLLTable`` whose only responsibility is to provide methods for accessing the table (basically it would just provide the relevant methods that are currently attached to ``EnzoInitialM1Closure``).
* The idea would be to just initialize an instance of this class in the constructor of ``EnzoMethodM1Closure`` and store it as an attribute of ``EnzoMethodM1Closure``.
* Doing this would keep the logic for reading and accessing the file separate from the rest of the logic of the solver (which might be helpful to people unfamiliar with the solver).

I just want to emphasize that this is very much a suggestion (in my very subjective opinion it makes the code "cleaner") - **please don't feel obliged to do it this**